manifests/manifest-beta.yaml

```diff
@@ -17,11 +17,11 @@ applications:
 - name: weathergov-beta
   <<: *defaults
   memory: 256M
-  instances: 2
+  instances: 1
   random-route: false
-- name: cronish
-  <<: *defaults
-  no-route: true
-  command: ./cronish.sh
-  health-check-type: process
-  memory: 128M
+# - name: cronish
+#   <<: *defaults
+#   no-route: true
+#   command: ./cronish.sh
+#   health-check-type: process
+#   memory: 128M
```

Comment:
[non-blocking question] can this commented-out code be removed? (same question for all manifest files)

Reply:
It's something we definitely want in the future but it's not something we are using currently.